src/sentry/conf/server.py

```diff
@@ -1141,7 +1142,14 @@ def create_partitioned_queues(name):
 # The cache version affects both Django's internal cache (at runtime) as well
 # as Sentry's cache. This automatically overrides VERSION on the default
 # CACHES backend.
-CACHE_VERSION = 1
+CACHE_VERSION = 2 if six.PY3 else 1
+
+# This setting modifies the cache version for `RedisClusterAttachmentCache`.
+# The attachment cache is not actually a cache, but a short term data storage system.
+# It inherits from cache backends, and so we're naming this setting to match this.
+# Bumping this cache version will likely result in dropping events, so be very careful
+# if you're doing so.
+ATTACHMENT_CACHE_VERSION = 1
```

Comment:
We could potentially just make this a constant in src/sentry/attachments/redis.py, not sure it needs to be configurable?

Reply:
Agreed, there's little point in versioning it like this. Mid-term goal is to get rid of the attachment cache completely and write to file store right away.

Just curious, what makes you say that this is not a cache (Oxford: "an auxiliary memory from which high-speed retrieval is possible")? A major part of these attachment is still dropped after processing.

Reply:
Typically a cache will replicate data from a persistent source for performance reasons, and losing that data won't result in data loss. Since we can't recalculate this data from other systems easily I'm not sure that this is a cache?